src/main/resources/messages.properties

```diff
@@ -42,5 +42,6 @@ patient.last.name.notnull = Patient's Last Name is a mandatory field
 patient.name.cannot.be.empty = The patient's name cannot be empty. Please provide a valid name.
 patient.name.not.found = No patients found with the given name.
 patient.not.found = Patient not found.
+user.not.authorized = You are not authorized to access this resource
```

Comment:
change this message to 
You do not have permission to access this resource.

Reply:
Resolved